### PR TITLE
Incomplete French Translation

### PR DESCRIPTION
**Pull request by Methylnaltrexone**

### CODE REVIEW

**Comment by antoine62** [comment]
Ok, so i think i need to complete it !

File: assets/romfs/i18n/fr/gui.json [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+    "LANGUAGE": "Langage",
+    "DEFAULTS": "Defaults", 
+    "MISC": "Div.",
+    "TID": "TID",
+    "SID": "SID",
+    "OT": "OT",
+    "NATIONALITY": "Nationalité",
+    "DAY": "Jour",
+    "MONTH": "Mois",
+    "YEAR": "Année,
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: `"` missing, causes an invalid json format

File: assets/romfs/i18n/fr/gui.json [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+    "ORIGINAL_TRAINER_FRIENDSHIP": "Original Trainer Friendship",
+    "ORIGINAL_TRAINER_AFFECTION": "Original Trainer Affection",
+    "ORIGINAL_TRAINER_MEMORY_INTENSITY": "Original Trainer Memory Intensity",
+    "ORIGINAL_TRAINER_MEMORY_LINE": "Original Trainer Memory Line",
+    "ORIGINAL_TRAINER_MEMORY_TEXTVAR": "Original Trainer Memory TextVar",
+    "ORIGINAL_TRAINER_MEMORY_FEELING": "Original Trainer Memory Feeling",
+    "EGG_RECEIVED_DATE": "Date de Réception de L'oeuf",
+    "MET_DATE": "Date de Rencontre",
+    "EGG_LOCATION": "Location de l'Oeuf",
+    "MET_LOCATION": "Location de Rencontre,
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: `"` missing, causes an invalid json format

File: assets/romfs/i18n/fr/gui.json [hunk 3]
```diff
@@ -0,0 +1,397 @@
+{
+    "LOADER_INSTRUCTIONS_TOP_PRESENT": "Choisir une sauvegarde à ouvrir. Presser Y pour les jeux présents.",
+    "LOADER_INSTRUCTIONS_TOP_ABSENT": "Choisir une sauvegarde à ouvrir. Presser Y pour les jeux absents.",
+    "LOADER_INSTRUCTIONS_BOTTOM":"Déplacez-vous avec le DPAD. A pour valider, START pour quitter",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Missing space after `:`

**Comment by BernardoGiordano** [comment]
Please include https://github.com/FlagBrew/PKSM/blob/PKSM.cpp/assets/romfs/i18n/en/gui.json#L397-L412 and https://github.com/FlagBrew/PKSM/blob/PKSM.cpp/assets/romfs/i18n/en/gui.json#L31

**Comment by BernardoGiordano** [comment]
Please translate the lines that I just added

File: assets/romfs/i18n/fr/gui.json [hunk 4]
```diff
@@ -0,0 +1,445 @@
+{
+    "LOADER_INSTRUCTIONS_TOP_PRESENT": "Choisir une sauvegarde à ouvrir. Presser Y pour les jeux présents.",
+    "LOADER_INSTRUCTIONS_TOP_ABSENT": "Choisir une sauvegarde à ouvrir. Presser Y pour les jeux absents.",
+    "LOADER_INSTRUCTIONS_BOTTOM": "Déplacez-vous avec le DPAD. A pour valider, START pour quitter",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string

File: assets/romfs/i18n/fr/gui.json [hunk 5]
```diff
@@ -0,0 +1,445 @@
+{
+    "LOADER_INSTRUCTIONS_TOP_PRESENT": "Choisir une sauvegarde à ouvrir. Presser Y pour les jeux présents.",
+    "LOADER_INSTRUCTIONS_TOP_ABSENT": "Choisir une sauvegarde à ouvrir. Presser Y pour les jeux absents.",
+    "LOADER_INSTRUCTIONS_BOTTOM": "Déplacez-vous avec le DPAD. A pour valider, START pour quitter", 
+    "LOADER_GAME_CARD": "Carte de Jeu",
+    "LOADER_INSTALLED_GAMES": "Jeux Installés",
+    "LOADER_ID": "ID: ",
+    "LOADER_MEDIA_TYPE": "Type de Media",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string

File: assets/romfs/i18n/fr/gui.json [hunk 6]
```diff
@@ -0,0 +1,445 @@
...
+    "DEFAULTS": "Défauts", 
+    "MISC": "Divers",
+    "TID": "TID",
+    "SID": "SID",
+    "OT": "DO",
+    "NATIONALITY": "Nationalité",
+    "DAY": "Jour",
+    "MONTH": "Mois",
+    "YEAR": "Année",
+    "CONFIG_BACKUP_SAVE": "Créer une sauvegarde automatiquement au lancement",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string a lot

File: assets/romfs/i18n/fr/gui.json [hunk 7]
```diff
@@ -0,0 +1,445 @@
...
+    "WC_DUMP1": "Appuyez sur \uE000 pour dump la CM",
+    "WC_INST1": "Presser \uE000 pour continuer ou \uE001 pour revenir en arrière.",
+    "EVENT_DATABASE": "Base de Données d'Évènements",
+    "FAILED_OPEN_DUMP": "Impossible d'ouvrir le fichier pour le dump !",
+    "SPATK.": "Atq. Spé.",
+    "SPDEF.": "Déf. Spé.",
+    "NEUTRAL": "Neutre",
+    "STORAGE": "Banque",
+    "EDITOR": "Editeur",
+    "EVENTS": "Evenements",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string a little bit

File: assets/romfs/i18n/fr/gui.json [hunk 8]
```diff
@@ -0,0 +1,445 @@
...
+    "ATTACK": "Attaque",
+    "DEFENSE": "Défense",
+    "SPATK":"Attaque spéc.",
+    "SPDEF": "Défense spéc.",
+    "SPEED": "Vitesse",
+    "EDITOR_HIDDEN_POWER": "Puissance Cachée",
+    "MOVES": "Capacités",
+    "RELEARN_MOVES": "Capacités réapprenables",
+    "EDITOR_NOT_APPLICABLE_GEN": "Pas applicable pour cette génération",
+    "EDITOR_INST": "Presser \ue000 pour sélectionner, \ue001 pour retourner en arrière",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: replace ` ` after `sélectionner,` with `\n`

File: assets/romfs/i18n/fr/gui.json [hunk 9]
```diff
@@ -0,0 +1,445 @@
...
+    "AWAKENED": "Awakened",
+    "EV": "EV",
+    "TOTAL": "TOTAL",
+    "HP": "PV",
+    "ATTACK": "Attaque",
+    "DEFENSE": "Défense",
+    "SPATK":"Attaque spéc.",
+    "SPDEF": "Défense spéc.",
+    "SPEED": "Vitesse",
+    "EDITOR_HIDDEN_POWER": "Puissance Cachée",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Add ` ` after `Cachée`

File: assets/romfs/i18n/fr/gui.json [hunk 10]
```diff
@@ -0,0 +1,445 @@
...
+    "NEUTRAL": "Neutre",
+    "STORAGE": "Banque",
+    "EDITOR": "Editeur",
+    "EVENTS": "Evenements",
+    "SCRIPTS": "Scripts",
+    "BAG": "Sac",
+    "SETTINGS": "Paramètres",
+    "SAVE_CHANGES_1": "Voulez-vous sauvegarder",
+    "SAVE_CHANGES_2": "Vos modifications ?",
+    "SCRIPTS_INST1": "Presser \uE000 Pour exécuter un script ou entrer dans un dossier. Presser \uE003 pour les scripts universel.",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string

File: assets/romfs/i18n/fr/gui.json [hunk 11]
```diff
@@ -0,0 +1,445 @@
...
+    "SCRIPTS_INST2": "Presser \uE002 pour changer entre les scrits intégrés et ceux sur la carte SD.",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string

File: assets/romfs/i18n/fr/gui.json [hunk 12]
```diff
@@ -0,0 +1,445 @@
...
+    "TOXIC": "Poison grave",
+    "HIDDEN_ABILITY?": "Talent Caché?",
+    "NS_POKEMON?": "Pokemon de N?",
+    "HEX_SELECTED_BYTE": "Octet Selectionné:",
+    "WC_LGPE": "Les sauvegardes de LGPE ne stockent pas de CM",
+    "NOT_A_BUG": "Ceci n'est pas un bug.",
+    "WC_SWITCH": "\uE004 / \uE005 pour changer entre différentes CM.",
+    "LANGUAGES": "Langues",
+    "OVERWRITE_WC": "Écraser Carte Miracle",
+    "ADAPT_LANGUAGE": "Adapter la langue du jeu à la CM",
```
Comment (BernardoGiordano) [inline assets/romfs/i18n/fr/gui.json]: Shorten this string

**Pull request merged**